about/faq.rst

```diff
@@ -622,7 +628,8 @@ for the rest of the game.
 How can I support Godot development or contribute?
 --------------------------------------------------
 
-See :ref:`doc_ways_to_contribute`.
+See :ref:`doc_ways_to_contribute`, or click "Edit on GitHub" on the top-right to add
```

Comment:
I'm not against this kind of call to action to edit the docs directly, but it's probably better suited to be on the page itself, not in the FAQ linking to the page.